modules/ducktests/tests/ignitetest/tests/discovery_test.py

```diff
@@ -139,24 +179,33 @@ def __basic_test__(self, version, with_zk=False):
 
         data["node_disco_info"] = disco_infos
 
-        self.servers.stop_node(fail_node, clean_shutdown=False)
-
         start = self.monotonic()
-        self.servers.await_event_on_node("Node FAILED", random.choice(survived_node), 60, True)
+
+        ids_to_wait = [node.discovery_info().node_id for node in failed_nodes]
+
+        self.servers.stop_nodes_async(failed_nodes, clean_shutdown=False, delay_ms=delay_ms)
+
+        for failed_id in ids_to_wait:
+            self.stage("Waiting for stopping " + failed_id)
+
+            self.servers.await_event_on_node("Node FAILED: TcpDiscoveryNode \\[id=" + failed_id, survived_node, 30,
+                                             from_the_beginning=True)
 
         data['Failure of node detected in time (s)'] = self.monotonic() - start
```

Comment:
May be change this report string? Write how many nodes failed?

Reply:
Fixed